filters/builtin/compress_test.go

```diff
@@ -270,14 +275,14 @@ func TestCompress(t *testing.T) {
 			"Content-Encoding": []string{"gzip"},
 			"Vary":             []string{"Accept-Encoding"}},
 	}, {
-		"does not accept encoding",
+		"does not accept encodingPriority",
 		http.Header{},
 		3 * 8192,
 		nil,
 		"",
 		http.Header{},
 	}, {
-		"unknown encoding",
+		"unknown encodingPriority",
```

Comment:
Lets revert encoding->encodingPriority changes in strings and docs: the `encodingPriority` is an internal structure name while e.g. this test verifies that unknown encoding has no effect, i.e. its not related to "priority" in any way

Reply:
good catch, bad refactoring. done